src/hooks/useVotesCountByDate.ts

```diff
@@ -13,8 +14,15 @@ export default function useVotesCountByDate(start: Date, end: Date) {
     staleTime: DEFAULT_QUERY_STALE_TIME,
   })
 
+  const parsedVotes = votes ? createMultipleVotes(votes) : undefined
+  const { highQualityVotes } = getMultipleVotesSegmentation(parsedVotes)
+
+  const votesCount = Object.values(highQualityVotes).reduce((acc, votes) => {
+    return acc + votes.length
+  }, 0)
+
   return {
-    votesCount: votes?.length,
+    votesCount: votes ? votesCount : undefined,
```

Comment:
Do we need `votesCount` to be undefined? If highQualityVotes is an empty array, votesCount will return 0.